tests/Backend/Workspaces/SynapseWorkspacesUnloadTest.php

```diff
@@ -95,6 +95,56 @@ public function testCreateTableFromWorkspaceWithInvalidColumnNames()
         }
     }
 
+    public function testImportFromWorkspaceWithInvalidTableName()
```

Comment:
Tenhle test by měl být pro všechny backendy.

Reply:
Možná by se dal napsat obecně prto všechny backendy najednou do obecného Bucket backend testu

Reply:
Pridal jsem test, ale otestoval jsem to jen an Synapse - ostatni nemam
Ale ty ostatni backendy se budou chovat jinak, tak by to chtelo nejak osetrit...

Reply:
Na Snowflaku to pada na:
```
    "\u0000*\u0000message": "odbc_prepare(): SQL error: SQL compilation error:\nTable \u0027MJ_KBC_4.WORKSPACE_605.\u0022test.LanguagesMissing\u0022\u0027 does not exist or not authorized., SQL state S0002 in SQLPrepare",
    "\u0000Exception\u0000string": "",
    "\u0000*\u0000code": 0,
    "\u0000*\u0000file": "\/var\/www\/html\/legacy-app\/vendor\/keboola\/php-csv-db-import\/src\/Snowflake\/Connection.php",
    "\u0000*\u0000line": 187,
    "\u0000Exception\u0000trace": [
        {
            "function": "{closure}",
            "class": "Bootstrap",
            "type": "-\u003E",
            "args": [
                2,
                "odbc_prepare(): SQL error: SQL compilation error:\nTable \u0027MJ_KBC_4.WORKSPACE_605.\u0022test.LanguagesMissing\u0022\u0027 does not exist or not authorized., SQL state S0002 in SQLPrepare",
                "\/var\/www\/html\/legacy-app\/vendor\/keboola\/php-csv-db-import\/src\/Snowflake\/Connection.php",
                187,
                {
                    "sql": "SHOW COLUMNS IN \u0022WORKSPACE_605\u0022.\u0022test.LanguagesMissing\u0022",
                    "bind": []
                }
            ]
        },
```

Reply:
Tenhle test case teda vyhodit ještě. Testuje se to obecně pro všechny (s tím, že ostatní se teď ignorují)